inlong-sort/sort-formats/format-common/src/main/java/org/apache/inlong/sort/formats/util/StringUtils.java

```diff
@@ -450,11 +489,11 @@ public static String[][] splitCsv(
                     result[i] = lines.get(i);
                 }
                 return result;
-
-            case STATE_ESCAPING:
-                throw new IllegalArgumentException(String.format("Not closed escaping. Text=[%s].", text));
-            case STATE_QUOTING:
-                throw new IllegalArgumentException(String.format("Not closed quoting. Text=[%s].", text));
+//
+//            case STATE_ESCAPING:
+//                throw new IllegalArgumentException(String.format("Not closed escaping. Text=[%s].", text));
+//            case STATE_QUOTING:
+//                throw new IllegalArgumentException(String.format("Not closed quoting. Text=[%s].", text));
```

Comment:
It's better to remove these lines of code if they are not used.

Reply:
done